### DISCUSSION TITLE
about disabled link jmalloc

### DISCUSSION THREAD

**Discussion by xubobbs** [duckdb/duckdb#13892] [Q&A]
I want to disabled jmalloc use .and the old version I can use -DSKIP_EXTENSIONS="jemalloc" to skip this .but the lastest version,I use this command compile and also see "
```

-- Extensions will be deployed to: /home/xubo/go/src/testcode/duckdb/duckdbLib/duckdb/build/release/repository
-- Load extension 'parquet' from '/home/xubo/go/src/testcode/duckdb/duckdbLib/duckdb/extensions' @ v1.1.0
-- Load extension 'jemalloc' from '/home/xubo/go/src/testcode/duckdb/duckdbLib/duckdb/extensions' @ v1.1.0
-- Extensions linked into DuckDB: [parquet, jemalloc]
```
and at last linked failed with error like this
```
./3rd/duck_db/x86/libduckdb.a(jemalloc.c.o): In function `malloc_init_hard':
jemalloc.c:(.text.unlikely+0x3d84): undefined reference to `pthread_atfork'
```
what should I do to not use jmalloc ?
thanks

**Comment by carlopi** [comment]
I would expect at the Makefile level:
```
SKIP_EXTENSIONS=jemalloc make
```
or equivalent at the CMake level that:
```
cd build/release
cmake -G "Ninja" -DSKIP_EXTENSIONS="jemalloc" -DCMAKE_BUILD_TYPE=Release ../..
cmake --build . --config Release
```
to compile with NO jemalloc AND also to not show the line `Load extension 'jemalloc'` line.

I just tried on native duckdb, in the latest main branch or in `v1.1.0`, and behaves as expected.

I am not sure if there are other variables at play, if you could share a more complete reproduction / logs it would be helpful.

**Comment by xubobbs** [comment]
thanks. now I use your shell,it output like 
```

-- Extensions will be deployed to: /home/xubo/go/src/testcode/duckdb/duckdbLib/duckdb/repository
-- Load extension 'parquet' from '/home/xubo/go/src/testcode/duckdb/duckdbLib/duckdb/extensions' @ v1.1.0
-- Extensions linked into DuckDB: [parquet]
```
no jemalloc again .thanks
